spec/customer_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'customer'
+
+describe Customer do
+
+  let(:takeaway) { double(:takeaway, menu: {:tomato_soup => 5,
```

Comment:
Style/Blocks: Avoid using {...} for multi-line blocks.<br>Style/HashSyntax: Use the new Ruby 1.9 hash syntax.<br>Style/MultilineBlockLayout: Block body expression is on the same line as the block start.